marathon_acme/tests/test_fake_vault.py

```diff
@@ -206,6 +206,36 @@ def test_create_kv_with_cas_mismatch(self):
         # No data set since CAS didn't match
         assert_that(data, Is(None))
 
+    def test_pre_create_update(self):
+        """
+        When a request is made to update KV data for and a pre-create/update
+        callback has been set, that callback is called before the request is
+        processed and afterwards the request proceeds as usual.
```

Comment:
"KV data for and a pre-create/update" -- I think the "for" in there needs to go.